python/ray/_raylet.pyx

```diff
@@ -261,9 +261,19 @@ cdef increase_recursion_limit():
     """Double the recusion limit if current depth is close to the limit"""
     cdef:
         CPyThreadState * s = <CPyThreadState *> PyThreadState_Get()
-        int current_depth = s.recursion_depth
         int current_limit = Py_GetRecursionLimit()
         int new_limit = current_limit * 2
+        cdef extern from *:
```

Comment:
Could you give me some pointers for this deprecation? Tried searching but couldn't find anything related.

Reply:
https://docs.python.org/3/whatsnew/3.11.html

> recursion_depth: removed, use (tstate->recursion_limit - tstate->recursion_remaining) instead.

Reply:
Ah nvm - got it.